packages/flutter/lib/src/material/carousel.dart

```diff
@@ -1261,7 +1261,8 @@ class _CarouselMetrics extends FixedScrollMetrics {
 
   /// The fraction of the viewport that the first item occupies.
   ///
-  /// Used to compute [item] from the current [pixels].
+  /// Used to compute the extent of each carousel item from the current [pixels],
```

Comment:
@QuncCccccc could you take a look and see if this makes sense?

Reply:
Yes, this makes sense to me. Thanks for also fixing the typos:)